pkg/readiness/ready_tracker.go

```diff
@@ -44,6 +45,8 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var ignoreFailedListing = flag.Bool("ignore-failures-when-fetching-expectations", false, "Determines whether the readiness tracker's Run method will fail-open (default) or fail-close when failing to fetch expectations")
```

Comment:
The current docstring for this flag requires the user to understand G8r internals to understand the comment. I think we can avoid this. Something like:

`When set (defaults to true), gatekeeper will ignore errors gathering expectations. This prevents bootstrapping errors from crashing Gatekeeper at the cost of increasing the risk Gatekeeper will under-enforce policy by serving before it has loaded in all policies. Disabling will help prevent under-enforcement at the risk of crashing during startup for issues like network errors.`

Feel free to simplify that, if possible (it's a mouthful).

Also, to maintain backwards compatibility, we should set the default value to `true`

Reply:
Should I instead name the variable dontIgnoreFailListing and default to false (I think we had a conversation about defaulting to true vs. false.)? For now I will make the changes as you mentioned but I can change it to this if we want.

Reply:
Oh, right. Let's make it default to false. We should use a variable name that avoids a double-negative, though.

Reply:
Done!